backend/graphql/resolvers/testResolvers.ts

```diff
@@ -171,6 +178,12 @@ const testResolvers = {
       });
       return updatedTest;
     },
+    uploadTestImage: async (
```

Comment:
nit: can we call `uploadQuestionImage`, since we use `question` terminology everywhere else in the editor?